src/lter/hbef/retrieve.R

```diff
@@ -57,14 +56,14 @@ for(i in 1:nrow(prod_info)){
             next
         }
 
-        if(! is.na(prod_info$munge_status[i])){
-            update_data_tracker_m(network = network,
-                                  domain = domain,
-                                  tracker_name = 'held_data',
-                                  prodname_ms = prodname_ms,
-                                  site_name = site_name,
-                                  new_status = 'pending')
-        }
+        #if(! is.na(prod_info$munge_status[i])){
```

Comment:
if you highlight a chunk and hit cmd+shift+c, it should neatly comment the whole chunk